include/librealsense2/h/rs_types.h

```diff
@@ -280,53 +280,6 @@ const char* rs2_get_failed_args                (const rs2_error* error);
 const char* rs2_get_error_message              (const rs2_error* error);
 void        rs2_free_error                     (rs2_error* error);
 
-#pragma pack(push, 1)
-/* rs2_calibration_roi - Array of four corners in Deph Frame Coordinate system that define a closed simple quadrangle (non-intersecting)*/
-typedef struct rs2_calibration_roi
-{
-    uint16_t mask_pixel[4][2];
-}rs2_calibration_roi;
-
-typedef struct float3_row_major { float x, y, z; } float3_row_major;
-typedef struct float3x3_row_major { float3_row_major x, y, z; } float3x3_row_major;
-
-typedef struct rs2_extrinsics_row_major
-{
-    float3x3_row_major rotation; // Rotation matrix
-    float3_row_major translation; // Metric units
-} rs2_extrinsics_row_major;
-
-typedef struct rs2_calibration_config
-{
-    uint8_t calib_roi_num_of_segments; // Within 0-4 range: 0 - Default.No limitations.Full FOV can be used in TC
-                                       //                   1 - 4: Segments defined.The segment must be sequential
-    rs2_calibration_roi roi[4];        // Segment 0 = convex tetragon - The vertices of the tetragon are ordered clockwise
-                                       //       Vertex = [x, y] = pixel coordinates in the reference depth map
-                                       //       0 - based coordinates : [0, 0] = center of the top - left pixel
-                                       // Segments 1-3 - structured identical to segment_#0 (reserved)
-                                       // The ROI segments can intersect, but each must be convex(angles <= 180 degrees).
-    uint8_t reserved1[12];
-    rs2_extrinsics_row_major camera_position;
-    uint8_t reserved2[300];
-    uint8_t crypto_signature[32];
-    uint8_t reserved3[39];
-} rs2_calibration_config;
-
-typedef struct rs2_calibration_config_header
-{
-    uint16_t version;       // major.minor. Big-endian
-    uint16_t table_type;    // type
-    uint32_t table_size;    // full size including: header footer
-    uint32_t calib_version; // major.minor.index
-    uint32_t crc32;         // crc of all the data in table excluding this header/CRC
-} rs2_calibration_config_header;
-
-typedef struct rs2_calibration_config_with_header
```

Comment:
This struct is not in used in the other branches, because the same code will be also deleted from there.
The struct rs2_extrinsics_row_major is needed in another branch and will need to be added in another place - please see ticket for details.